pkg/provider/kubernetes/crd/traefikio/v1alpha1/service.go

```diff
@@ -86,3 +88,12 @@ type WeightedRoundRobin struct {
 	// More info: https://doc.traefik.io/traefik/v3.5/routing/providers/kubernetes-crd/#stickiness-and-load-balancing
 	Sticky *dynamic.Sticky `json:"sticky,omitempty"`
 }
+
+// +k8s:deepcopy-gen=true
+
+// HighestRandomWeight holds the highest radom weight configuration.
+// More info: https://doc.traefik.io/traefik/v3.5/routing/services/#highest-random-configuration
+type HighestRandomWeight struct {
+	// Services defines the list of Kubernetes Service and/or TraefikService to load-balance, with weight.
+	Services []Service `json:"services,omitempty"`
+}
```

Comment:
There's also `HealthCheck` in the dynamic configuration.
Could you please fix this inconsistency between file-based and CRD-based configurations?